pkgs/development/libraries/jemalloc/default.nix

```
@@ -19,6 +20,14 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-LbgtHnEZ3z5xt2QCGbbf6EeJvAU3mDw7esT3GJrs/qo=";
   };
 
+  patches = [
+    # fix tests under --with-jemalloc-prefix=, see https://github.com/jemalloc/jemalloc/pull/2340
+    (fetchpatch {
+      url = "https://github.com/jemalloc/jemalloc/commit/d00ecee6a8dfa90afcb1bbc0858985c17bef6559.patch";
+      hash = "sha256-11wp6byrhxi8ymqgknb40xx5xcwyj60ydajqh2029qc924ivi61p";
```

Terribly sorry for the wrong suggestion, this needs to be proper SRI-hash for `hash` to work of course.

```suggestion
      hash = "sha256-N5i4IxGJ4SSAgFiq5oGRnrNeegdk2flw9Sh2mP0yl4c=";
```